Mathlib/MeasureTheory/Measure/FiniteMeasure.lean

```diff
@@ -274,15 +274,29 @@ theorem restrict_nonzero_iff (μ : FiniteMeasure Ω) (A : Set Ω) : μ.restrict
 
 variable [TopologicalSpace Ω]
 
-/-- Two finite Borel measures are equal if the integrals of all bounded continuous functions with
-respect to both agree. -/
+/-- Two finite Borel measures are equal if the integrals of all non-negative bounded continuous
+functions with respect to both agree. -/
 theorem ext_of_forall_lintegral_eq [HasOuterApproxClosed Ω] [BorelSpace Ω]
     {μ ν : FiniteMeasure Ω} (h : ∀ (f : Ω →ᵇ ℝ≥0), ∫⁻ x, f x ∂μ = ∫⁻ x, f x ∂ν) :
     μ = ν := by
   apply Subtype.ext
   change (μ : Measure Ω) = (ν : Measure Ω)
   exact ext_of_forall_lintegral_eq_of_IsFiniteMeasure h
 
+/-- Two finite Borel measures are equal if the integrals of all bounded continuous functions with
+respect to both agree. -/
+theorem ext_of_forall_integral_eq [HasOuterApproxClosed Ω] [BorelSpace Ω]
+    {P P' : FiniteMeasure Ω} (h : ∀ (f : Ω →ᵇ ℝ), ∫ x, f x ∂P = ∫ x, f x ∂P') :
```

Comment:
Measures are called `μ ν` in that file. Could you use the same names for this lemma?